source/cursorManager.py

```diff
@@ -1,8 +1,8 @@
-#cursorManager.py
-#A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2006-2018 NV Access Limited, Joseph Lee, Derek Riemer, Davy Kager
-#This file is covered by the GNU General Public License.
-#See the file COPYING for more details.
+# cursorManager.py

```

Comment:
@CyrilleB79 I wondered about the file name in the header, but since I didn't put it there, I wasn't sure if it was reasonable for me to be the one to remove it. I assumed the article referred to new headers. Is it customary for one to correct this when simply updating the header?

Reply:
>Is it customary for one to correct this when simply updating the header?

Yes, I was about to suggest the same change as Cyrille